ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java

```diff
@@ -14,6 +17,7 @@
     private static final By FIRST_CONDITION_NAME = By.xpath("//table/tbody[2]/tr[1]/td[1]");
     private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
     private static final By DELETE = By.cssSelector("i[title='Delete']");
+    private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
```

Comment:
this list should also be referenced with a more reliable referencing mechanism. consider parent elements that would qualify any sub elements